stdlib/public/stubs/LibcShims.cpp

```diff
@@ -13,14 +13,34 @@
 #if defined(__APPLE__)
 #define _REENTRANT
 #include <math.h>
+#include <CoreFoundation/CoreFoundation.h>
```

Comment:
Is this kosher for the standard library? We could do without the detailed error message, IMO, if that simplifies dependencies.

Reply:
@xwu This [dependency](https://github.com/apple/swift/blob/6cf1aa308344a26760fbb803ed7c409a451c2ad6/stdlib/public/core/CMakeLists.txt#L188) already exists. But I could remove the `SecCopyErrorMessageString` call.